app/assets/javascripts/discourse/app/components/user-preferences/categories.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="control-group category-notifications">
   <label class="control-label">{{i18n "user.categories_settings"}}</label>
 
-  <div class="controls tracking-controls tracking-controls--watched-categories">
+  <div class="controls tracking-controls tracking-controls__watched-categories">
```

Comment:
I'm following https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/ which means this should be using `__` to represent an element. `--` is reserved for state.

Reply:
Yes this is correct :+1: